README.md

```diff
@@ -1,21 +1,113 @@
 # ogcapi-operator
 
-A controller/operator that manages the Kubernetes resources to serve an OGC API according to specification.
+_Kubernetes controller/operator to serve an OGC API according to spec._
+
+[![Build](https://github.com/PDOK/ogcapi-operator/actions/workflows/build-and-publish-image.yml/badge.svg)](https://github.com/PDOK/ogcapi-operator/actions/workflows/build-and-publish-image.yml)
+[![Lint (go)](https://github.com/PDOK/ogcapi-operator/actions/workflows/lint-go.yml/badge.svg)](https://github.com/PDOK/ogcapi-operator/actions/workflows/lint-go.yml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/PDOK/ogcapi-operator)](https://goreportcard.com/report/github.com/PDOK/ogcapi-operator)
+[![Coverage (go)](https://github.com/PDOK/ogcapi-operator/wiki/coverage.svg)](https://raw.githack.com/wiki/PDOK/ogcapi-operator/coverage.html)
```

Comment:
De coverage geeft nu vertekend beeld (te laag). Ik zou `-coverpkg` aan `go test` toevoegen in de Makefile.

Reply:
ik kreeg dat zo snel niet voor elkaar met -coverpkg in make test omdat iets dan alsnog de coverage probeert uit te rekenen met tmp files die het niet kan vinden. ik heb nu maar even de grep filter uitgebreid op de cover.out.tmp in de github action